recipes/virusrecom/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set name = "virusrecom" %}
+{% set version = "1.1.2.2" %}
+
+package:
+  name: "{{ name }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: d06eb57299eb38b3fbf2341af3ede2265a400962c2d147d84d19993ed496d024
+
+build:
+  number: 0
+  noarch: python
+  run_exports:
+    - {{ pin_subpackage('virusrecom', max_pin="x") }}
+
+
+requirements:
+  host:
+    - python >=3.5,<3.8
+    - pip
+    - setuptools
+
+  run:
+    - matplotlib-base
+    - numpy >=1.19.3
+    - pandas >=1.1.5
+    - scipy >=1.5.4
+    - psutil >=5.9.1
+    - openpyxl >=3.0.5
+
+test:
+  commands:
+    - virusrecom --help
+
+about:
+  home: "https://github.com/ZhijianZhou01/virusrecom"
+  license: "LGPL v2.1"
+  license_family: "GPL"
```

```suggestion
  run:
    - matplotlib-base
    - numpy >=1.19.3
    - pandas >=1.1.5
    - scipy >=1.5.4
    - psutil >=5.9.1
    - python >=3.5,<3.8
    - openpyxl >=3.0.5

test:
  commands:
    - virusrecom --help

about:
  home: "https://github.com/ZhijianZhou01/virusrecom"
  license: "LGPL-2.1-or-later"
  license_family: "LGPL"
```
Please add `python` to the list of run dependencies as well since the package is listed as `noarch: python`. Additionally, please use a [SPDX identifier ](https://spdx.org/licenses/) for the `license` field.